DiztinGUIsh/window/dialog/GotoDialog.Designer.cs

```diff
@@ -59,7 +60,7 @@ private void InitializeComponent()
             // textROM
             // 
             this.textROM.Location = new System.Drawing.Point(103, 12);
-            this.textROM.MaxLength = 6;
+            this.textROM.MaxLength = 8;
```

Comment:
I changed this size to allowing pasting of larger addresses, which means you can type in larger addresses than 24bit and the form won't stop you.

But, I added the validation checks which should serve the same purpose.